x-pack/plugins/alerting/server/lib/snooze/is_snooze_active.ts

```diff
@@ -54,8 +57,27 @@ export function isSnoozeActive(snooze: RuleSnoozeSchedule) {
   return null;
 }
 
-export function parseByWeekday(byweekday: Array<string | number>): ByWeekday[] {
+export function parseByWeekday(
+  byweekday: Array<string | number>,
+  dtstart: string,
+  tzid: string
+): ByWeekday[] {
   const rRuleString = `RRULE:BYDAY=${byweekday.join(',')}`;
   const parsedRRule = rrulestr(rRuleString);
-  return parsedRRule.origOptions.byweekday as ByWeekday[];
+  const newbyweekday = parsedRRule.origOptions.byweekday as Weekday[];
+  const startMomentDay = moment(dtstart).tz(tzid).isoWeekday();
+  const startMomentUTCDay = moment(dtstart).tz('UTC').isoWeekday();
+  if (startMomentDay !== startMomentUTCDay) {
+    return newbyweekday.map((d) => shiftWeekday(d, startMomentDay - startMomentUTCDay));
+  }
```

Comment:
Ok cool, from what I can understand we're taking their dtstart as a sample to see if they are + or - UTC and get the offset. Then we offset the `byweekday` by that much. 

:+1: